python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_nux.py

```diff
@@ -28,3 +29,16 @@ def test_stores_nux_seen_state(graphql_context):
     assert not result.errors
     assert result.data
     assert result.data["shouldShowNux"] is False
+
+
+def test_does_not_show_nux_if_read_only_filesystem(graphql_context):
+    def mock_seen_filepath():
+        fn = mock.Mock()
+        fn.side_effect = OSError("Read-only filesystem")
+        return fn
+
+    with mock.patch("dagster._core.nux.nux_seen_filepath", new_callable=mock_seen_filepath):
```

Comment:
this can be with mock.patch("dagster._core.nux.nux_seen_filepath", side_effect=OSError("Read-only filesystem")):

Reply:
til